### PR TITLE
Next

### PR DESCRIPTION
**Pull request by danilowoz**
https://dependency-seo-pages-next.vercel.app/packages/react

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,11 +1,86 @@
 {
```
Comment (danilowoz) [inline .eslintrc.json:1]: Got this from Codebox repo

**Comment by CompuIves** [comment]
One thing we'll need to do is change the url structure to be like this:

```
/examples/package/react
```

So that we can seamlessly replace the existing deploy, if possible 😄 .

File: .eslintrc.json (hunk 1 above)
Comment (CompuIves) [reply .eslintrc.json:1]: Nice!

**Comment by danilowoz** [comment]
Hey @CompuIves, as I'm using only the last slug of the path, it works either `examples/packages/react` or `packages/react`, however, in a second thought, maybe it would be better to fix the path. What do you think?

**Comment by CompuIves** [comment]
The sitemap generator assumes right now that the urls are on `/example/package/:package-slug`, so that's why it would be easier to go with such a route first. Here's a sitemap generated by it: https://codesandbox.io/sitemap/dependency/sitemap.0.xml. If it would not be possible with Next.js, we can maybe do some magic in our router to make it still work 😄

**Review by DannyRuchtie — CHANGES_REQUESTED** [review]
Looks great I spotted 3 things 
- Load more button renders incorrect in Safari
- We miss the environment icon (if we still want them there) 
- The fonts in the footer are different from the one on the homepage
This is also in the version that is on production right now but is a bug :D 

Screenshot 2021-05-19 at 17 18 56
Screenshot 2021-05-19 at 17 20 54

**Review by DannyRuchtie — APPROVED** [review]
looking good

**Review by CompuIves — APPROVED** [review]

**Pull request merged**